packages/bodiless-core/src/Store/BodilessMobxStore.static.ts

```diff
@@ -12,22 +12,14 @@
  * limitations under the License.
  */
 
-import {
```

Comment:
Hm - would another option here be to create static versions of the mobx imports rather than duplicating the store code itself?

Reply:
Actually I see you do this below for the StoreItem.... is there a reason you odn't do it here?